### DISCUSSION TITLE
Delete several explicit m:n relationship rows in update

### DISCUSSION THREAD

**Discussion by JoseFreitas990** [prisma/prisma#16204] [Q&A]
I've an explicit many-to-many relationship between User and Group.

I'm developing a controller in node which I give a group id and several users id's. The objective is to disconnect the relationship of this users with the group. (Delete the rows in the relational table)
The problem is that when I'm trying to use the disconnect, it asks me for the relational key (groupId_userId) when I wanted to put the userId only.

Maybe I should use another function but I'm not sure.

Here is the function:
```
async deleteUsersFromGroup(id: string, userData: DeleteGroupDto): Promise<any> {
    const response = prisma.group.update({
      where: {
        id: id,
      },
      data: {
        user: {
          disconnect: /* Trying to insert here the array of user's id  (userData) */
        },
      },
      include: { user: true },
    });

    return response;
  }
```

**Comment by nurul3101** [comment]
Seems this is related to this answered Stack Overflow Question: https://stackoverflow.com/questions/74376707/cant-disconnect-relation-of-explicit-many-to-many

Let me know if this is still an issue.

**Reply by pantharshit00** [reply]
👋 Hello @JoseFreitas990, please let us know if this is still relevant.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
